### ISSUE TITLE
Request for New Feature: Customizing Text and Footer in Report Emails

### ISSUE THREAD

**Issue by mavandigital** [piwik/piwik#20856]
Hello Matomo Team,

I hope this email finds you well. Firstly, I would like to express my gratitude for your excellent work on Matomo. I have found it extremely useful and well-designed, and it has become an indispensable tool in my daily work.

I am writing to discuss a potential new feature that I believe could further enhance the user experience on Matomo. I have noticed that, currently, there is no option to customize the text and footer in report emails. This customization ability could give users the flexibility to tailor their reports according to their specific needs and the audience they are intended for.

The addition of this feature would allow users to input personalized comments, notes, or conclusions in the body of the email message, allowing greater flexibility in the presentation of data. Additionally, customizing the footer could be used to include contact information, copyright notices, or any other information that an organization might want to include in their reports.

I understand that developing new features requires time and resources, but I believe this addition could bring significant value to many Matomo users. I would be happy to further discuss this proposal if you are interested, and I am open to providing any feedback or additional information that might aid in the evaluation of this request.

Thank you for your consideration and for your ongoing work on Matomo.

Best regards,

**Comment by michalkleiner** [comment]
Hi @mavandigital, thank you for the nice feedback and your kind words.

We appreciate the suggestion, it definitely sounds like a useful thing to be able to customise/add additional content to report emails. I'll pass this on to the product team for prioritisation.

**Comment by michalkleiner** [comment]
I will also close this issue for now as a partial duplicate and we can direct all the communication to https://github.com/matomo-org/matomo/issues/20294 which also has a hint how to customise some parts of the report.

**Comment by MatomoForumNotifications** [comment]
This issue has been mentioned on **Matomo forums**. There might be relevant details there:

https://forum.matomo.org/t/change-the-text-of-the-message-and-the-footer-of-the-matomo-report-email/51662/3

**Comment by Stan-vw** [comment]
Opening this as the other ticket is just a simple text change and already in the current sprint. This one needs a bit more design. Ideas on how it should work are welcome.

**Comment by atom-box** [comment]
From a user:

I make cheap WP websites (albeit with bespoke plugins and themes) for small family run businesses - guest houses, caravan sites, pubs, dog trainers, parish councils etc . My customers are (independent businesses owners, lone founders and entrepreneurs,) wondering if their website is worth the money it costs to host it. They need to see a very simple and clear report with lots of explanation.

I would like the option for a 'Dummies Guide' explanation of the report to appear below the data, either one you have written or the option for me to write them myself. Either way they would need to be global to all reports.

**Comment by atom-box** [comment]
From a user:

> Do you know, by any chance, if in the final version of 5.0 will be possible to customize the report that we send to the client (email report)?

**Comment by mattab** [comment]
@atom-box Yes, it's already possible in Matomo 4.x by setting a "Description" in the email report:

![image](https://github.com/matomo-org/matomo/assets/466765/2be35be8-edf4-430f-adf0-58498fa515fd)

**Comment by atom-box** [comment]
@mattab  

Yes it is possible currently. But it is inadequate:

" _would like the option for a 'Dummies Guide' explanation of the report to appear_ **below the data, either one you have written or the option for me to write them myself.** _Either way they would need to be global to all reports._"

The user wants more fields, longer fields that can hold an entire paragraph.

**Comment by atom-box** [comment]
Another user has requested better emailed reports:

> Question: What is your main goal for using Matomo?
> Answer:
> Be able to automate the sending of reports **correctly**. 
> Current reports are **not usable** in their current state.

**Comment by atom-box** [comment]
Another user, a large bank, has requested customization, this time in the individual sections. 

- In the title page of the report, we have a text saying "Date Range". Is it possible to change this to "Date"?
- In the table below Visits summary, there's a row with title "Avg Visit Duration (in seconds)" . Is it possible to change this to "Avg Visit Duration".
- In the performance overview section of the report, the graph has a title "Avg. page load time". Can this be changed to "Avg. Page load time (in seconds)".

(Sorry for the story creep but it seems logical. Maybe the title of this story could be Customizing Text, Footer, and Section Titles.)
